pyqt/pyqt5.json

```diff
@@ -0,0 +1,115 @@
+{
+    "name": "pyqt5",
+    "config-opts": [
```

Comment:
Please add `--debug` (generate builds with debuginfo) and `--verbose` (enable verbose/debug output during the build)